7_react/react1/week3/todo-list-app/src/EmployeesList.js

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from 'react';
+
+
+const Employees = () => {
+  const [data, setData] = useState([]);
+
+  const fetchData = async ()=>{
+
+    try {
+        let res = await fetch('https://gist.githubusercontent.com/benna100/5fd674171ea528d7cd1d504e9bb0ca6f/raw')
+        let data = await res.json()
+        setData(data)
+    }catch(error){
```

Comment:
I would highly recommend getting into the habit of formatting your code before committing, so the code looks neater. 
You can even set VS code to to this automatically every time you save or after every few seconds. 
See https://code.visualstudio.com/docs/editor/codebasics#_formatting